### PR TITLE
Ivander - edit/delete trip command

### PR DESCRIPTION
**Pull request by ivanderjmw**
- [x] Edit trip command
- [x] Delete trip command
- [x] Trip index optimization

### CODE REVIEW

File: src/main/java/seedu/trippie/data/TrippieData.java [hunk 1]
```diff
@@ -21,35 +27,33 @@ public TrippieData(Storage storage) {
...
+    }
 
-            if (maxDay == 0) {
-                list = list.concat(
-                        (trip.getIndex() + 1) + ". " + trip.getName() + " " + "[No places or expenses added yet]" + "\n"
-                );
-            } else {
-                boolean plural = maxDay != 1;
-                String dayWord = plural ? "Days" : "Day";
+    public Trip getTripFromIndex(int index) {
```
Comment (felixhalim) [inline src/main/java/seedu/trippie/data/TrippieData.java:39]: I like how you abstract this method!

**Review by felixhalim — APPROVED** [review]
Great job Ivander!

**Review by ShawnTanzc — APPROVED** [review]
Looking really solid, I will do some testing once it is merged! Thank you ivander!

**Review by tengkianen — APPROVED** [review]
lgtm!

**Pull request merged**